src/test/scala/scala/tools/refactoring/tests/analysis/CompilationUnitDependenciesTest.scala

```diff
@@ -792,7 +788,26 @@ class CompilationUnitDependenciesTest extends TestHelper with CompilationUnitDep
         @throws(classOf[IOException])
         def read() = in.read()
       }
-    """)
+    """
+    val deps1 = """java.io.BufferedReader
+         java.io.FileReader
+         java.io.IOException
+         scala.this.Predef.String
+         """
+    val deps2 = deps1 + "\n" + scala.this.Predef.classOf
```

Comment:
oh ok, you are right. There was only a compilation error because the `scala.this.Predef.classOf` is not part of the string literal.